ui/src/literature/containers/CiteAllActionContainer.jsx

```diff
@@ -1,11 +1,14 @@
 import { connect } from 'react-redux';
-import omit from 'lodash.omit';
 
+import omit from 'lodash.omit';
 import CiteAllAction from '../components/CiteAllAction';
 
 const stateToProps = state => ({
   numberOfResults: state.search.get('total'),
-  query: omit(state.router.location.query, ['sort', 'page']),
+  query: {
+    sort: state.search.getIn(['scope', 'query', 'sort']),
```

Comment:
maybe it would be better to add something like:
```js
// set default `sort` incase there is no `sort`, in the location query
```

Reply:
btw reading from the scope is OK, but since this is only for literature, alternatively you can directly read it from `searchScopes`.